docs/devdocs/architecture/data-flows/fhe-operation.md

```diff
@@ -17,7 +17,8 @@ This document outlines the complete flow of an FHE (Fully Homomorphic Encryption
 | **dApp** | The decentralized application that requests FHE operations |
 | **FHE.sol** | The library providing FHE operation functions |
 | **Task Manager** | Verifies and forwards operation requests |
-| **Aggregator** | Manages the request queue and communicates with the execution layer |
+| **Slim Listener** | Listens to chain events from Task Manager |
```

Comment:
```suggestion
| **Slim Listener** | Listens to chain events from the Task Manager Contract |
```

Otherwise it may seem that the Task manager is an off chain component such as the rest